### ISSUE TITLE
Main Training Loop for Embedding System

### ISSUE THREAD

**Issue by tomiock** [tomiock/TARS#7]
Notes: 
- `pytorch_metric_learning` provides miners and loss functions
- on a python file to execute remotely
- good logging (weights and biases)

Steps:
- [x] take the pre-made DataFrameS with the translator and task data
- [x] drop non necessary columns (e.g. translator_name or task_id)
- [x] do any necessary encodings (if not already done)
- [x] do any necessary scaling
- [x] create torch `Dataset` and `DataLoader`
- [x] create mining scheme
- [x] training loop using triplet loss
